test/configs/models/products.js

```diff
@@ -1,155 +1,204 @@
+  "roles": {
+    "create": [
+      "unauthenticated"
+    ],
+    "read": [
+      "unauthenticated"
+    ],
+    "update": [
+      "unauthenticated"
+    ],
+    "delete": [
+      "unauthenticated"
+    ]
   },
-  "chips": [
-    {
-      "verbose": "color",
-      "columns": [
-        {
-          "name": "color",
-          "verbose": "standard"
-        }
-      ]
-    }
-  ],
   "columns": [
     {
       "name": "id",
-      "verbose": "id",
-      "type": "int(11)",
-      "allowNull": false,
+      "searchable": true,
       "visible": {
-        "main": true,
-        "detail": true
+        "list": true,
+        "detail": true,
+        "relation": true
+      },
+      "model": {
+        "type": "int",
+        "allowNull": false,
+        "isPk": true
       },
-      "editable": true,
-      "input": {
-        "type": "number"
+      "layout": {
+        "label": "Id",
+        "listColumn": {},
+        "editField": {
+          "type": "number"
+        }
       }
     },
     {
       "name": "name",
-      "verbose": "name",
-      "type": "varchar(255)",
-      "allowNull": true,
+      "searchable": true,
       "visible": {
-        "main": true,
-        "detail": true
+        "list": true,
+        "detail": true,
+        "relation": false
+      },
+      "model": {
+        "type": "varchar(255)",
+        "allowNull": true
       },
-      "editable": true,
-      "input": {
-        "type": "text"
+      "layout": {
+        "label": "Name",
+        "listColumn": {},
+        "editField": {
+          "type": "text"
+        }
       }
     },
     {
       "name": "color",
-      "verbose": "color",
-      "type": "varchar(255)",
-      "allowNull": true,
+      "searchable": true,
       "visible": {
-        "main": true,
-        "detail": true
+        "list": true,
+        "detail": true,
+        "relation": false
       },
-      "editable": true,
-      "input": {
-        "type": "text"
-      }
-    },
-    {
-      "name": "active",
-      "verbose": "active",
-      "type": "tinyint(1)",
-      "allowNull": true,
-      "visible": {
-        "main": false,
-        "detail": true
+      "model": {
+        "type": "varchar(255)",
+        "allowNull": true
       },
-      "editable": true,
-      "input": {
-        "type": "checkbox"
+      "layout": {
+        "label": "Color",
+        "listColumn": {},
+        "editField": {
+          "type": "text"
+        }
       }
     },
     {
       "name": "type",
-      "verbose": "type",
-      "type": "smallint(5)",
-      "allowNull": true,
+      "searchable": true,
       "visible": {
-        "main": true,
-        "detail": true
+        "list": true,
+        "detail": true,
+        "relation": false
+      },
+      "model": {
+        "type": "smallint",
+        "allowNull": true
       },
-      "editable": true,
-      "input": {
-        "type": "number"
+      "layout": {
+        "label": "Type",
+        "listColumn": {},
```

Comment:
the default value on the config should be a boolean

Reply:
This is to be applied to all fields that take an object or a boolean has an input.